components/authz-service/server/v2/project_test.go

```diff
@@ -247,36 +247,78 @@ func TestDeleteProject(t *testing.T) {
 			assert.Nil(t, resp)
 		}},
 		{"fails with NotFound when deleting from empty store", func(t *testing.T) {
-			require.Zero(t, store.ItemCount())
+			require.Zero(t, projects.ItemCount())
 			resp, err := cl.DeleteProject(ctx, &api.DeleteProjectReq{Id: "test"})
 			grpctest.AssertCode(t, codes.NotFound, err)
 			assert.Nil(t, resp)
 		}},
 		{"deletes custom project when one project is in database", func(t *testing.T) {
-			require.Zero(t, store.ItemCount())
+			require.Zero(t, projects.ItemCount())
```

Comment:
let's update the test name to `deletes custom project with no roles when one project is in database`

Reply:
Updated